release/RELEASE.md

```diff
@@ -26,10 +26,22 @@ latest commit on master.
    ```
 
 1. Pull the latest master branch of the `daml` repository and create a new,
-   clean branch off it. For a snapshot, run `./release.sh snapshot HEAD`; for
-   a stable release, run `echo "$SHA $VERSION" > LATEST`. Ideally, for a stable
-   release, the resulting change is only to cut off the prerelease part of the
-   version number (the `-snapshot...`).
+   clean branch off it.
+
+   - For a snapshot run
+     - `./release.sh snapshot HEAD`;
+     - Edit `LATEST`, to update the release "version". For example, change
```

Comment:
This is only needed if the last release was a stable one; otherwise we do not want to increment the "stable" part. Maybe:
```suggestion
     - If applicable (e.g. latest release was a stable one), edit `LATEST` to update the release "version". For example, change
```